examples/jdbc-query-operation/jdbc_query_operation.bal

```diff
@@ -25,12 +25,6 @@ function simpleQuery(jdbc:Client jdbcClient) {
         io:println(e);
     }
 
-    // In general cases, the stream will be closed automatically
-    // when the stream is fully consumed or any error is encountered.
-    // However, in case if the stream is not fully consumed, the stream
-    // should be closed specifically.
-    e = resultStream.close();
```

Comment:
Any reason to remove this?

Reply:
Related to discussions in https://github.com/ballerina-platform/ballerina-distribution/issues/730